pkg/kubewarden/l10n/en-us.yaml

```
@@ -30,23 +30,25 @@ kubewarden:
     headers:
       policyServer:
         title: Policy Servers
-        description: Kubewarden uses Policy Servers to receive requests to be validated. It does that by executing Kubewarden's policies.
+        description: Policy Servers validate requests
         linkText: Create Policy Server
-        slotTitle: Policy Server Pods
       admissionPolicy:
-        title: Admission Policies
-        description: An Admission Policy is a namespace-wide resource. The policy will process only the requests that are targeting the Namespace where the Admission Policy is defined.
-        linkText: Create Admission Policy
-        slotTitle: Policies
+        title: Namespace Policies
```

Not sure if this it typo or correct :)
```suggestion
        title: Namespaced Policies
```